boards/shields/rtkmipilcdb00000be/Kconfig.defconfig

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_RTKMIPILCDB00000BE
+if DISPLAY
+
+config INPUT_INIT_PRIORITY
+	default 89
+
+# Enable MIPI DSI, as this display controller requires it.
+
+endif # DISPLAY
+
+if LVGL
+
+# Configure LVGL to use touchscreen with KSCAN API
+
+config INPUT
+	default y
+
+config INPUT_GT911_INTERRUPT
+	default n
+
+# LVGL should allocate buffers equal to size of display
+config LV_Z_VDB_SIZE
+	default 100
+
+# Enable double buffering
+config LV_Z_DOUBLE_VDB
+	default y
+
+# Force full refresh. This prevents memory copy associated with partial
```

Comment:
This comment is probably coming from this file, right?

https://github.com/zephyrproject-rtos/zephyr/blob/5dc5fa5ee2ac4c0b3914ce926d1e126f052d1e99/boards/shields/rk055hdmipi4m/Kconfig.defconfig#L31

If you need this to be enabled, update the comment to refer to the LCD controller on the Renesas board.